apps/web/src/app/vault/components/vault-items/vault-cipher-row.component.html

```diff
@@ -151,11 +156,25 @@
       <i class="bwi bwi-fw bwi-file-text" aria-hidden="true"></i>
       {{ "eventLogs" | i18n }}
     </button>
-    <button bitMenuItem (click)="restore()" type="button" *ngIf="cipher.isDeleted">
+    <button
+      bitMenuItem
+      (click)="restore()"
+      type="button"
+      *ngIf="
+        (limitItemDeletion$ | async)
+          ? cipher.isDeleted && cipher.permissions.restore && canDeleteCipher
+          : cipher.isDeleted && canDeleteCipher
```

Comment:
I think you're right, but let's avoid changing old code. If the feature flag is off, it should be the same as `ngIf` is today.

```suggestion
          : cipher.isDeleted
```